ProcessLib/ChemistryLib/PhreeqcInterfaceData/Solution.cpp

```diff
@@ -53,12 +72,21 @@ std::ofstream& operator<<(std::ofstream& outFile, Solution const& solution)
 
     outFile << "pressure " << solution._pressure << std::endl;
 
-    if (solution._charge_balance.empty())
-        outFile << "pH " << solution._pH << std::endl;
-    else
-        outFile << "pH " << solution._pH << " charge" << std::endl;
-
-    outFile << "pe " << solution._pe << std::endl;
+    switch (solution._means_of_adjusting_charge)
+    {
+        case MeansOfAdjustingCharge::pH:
+            outFile << "pH " << solution._pH << " charge" << std::endl;
+            outFile << "pe " << solution._pe << std::endl;
```

Comment:
Don't use `std::endl`, use `"\n"`.
Flushing the stream is not needed after every line, but is happening when using the `std::endl`.

Reply:
all replaced by "\n"